### DISCUSSION TITLE
Sqlite 3 types support on sqflite

### DISCUSSION THREAD

**Discussion by Temurxon77** [tekartik/sqflite#639] [Q&A]
### Good day to everyone, I have question regarding importing local .db asset from app and load to sqflite,currently I am creating sqflite 3 schema with NVarchar and other types after I will insert raw datas into the tables,can I load them in sqflite flutter app or I should change them to supported types on Sqflite ?thanks in advance for reply

**Comment by alextekartik** [comment]
I think NVarchar is just an alias to TEXT in sqlite. I don't know whether it will always be the case
